### PR TITLE
KAA-1502 Update links to github in the documentation

### PR DESCRIPTION
**Pull request by vadimol**
1) Added github_url_raw to refer raw files on github
2) Updated links to kaaproject github in the documentation
3) Updated "Documentation contribution and style guide"

### CODE REVIEW

File: doc/Administration-guide/Upgrading-your-instance/0.8.1-to-0.9.0/index.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ This guide describes how to upgrade your [Kaa server]({{root_url}}Glossary/#kaa-
 Follow these steps to run the upgrade script:
 
 1. Go to your Kaa host machine.
-2. Download the [upgradeKaa081to090.sh](https://github.com/kaaproject/kaa/blob/release-0.10/server/upgrade/data-migration-0.8.1-0.9.0/upgradeKaa081to090.sh), [upgradeMongo081to090KaaVersion.js](https://github.com/kaaproject/kaa/blob/release-0.10/server/upgrade/data-migration-0.8.1-0.9.0/upgradeMongo081to090KaaVersion.js), and [upgradePostgresql081to090kaaVersion.sql](https://github.com/kaaproject/kaa/blob/release-0.10/server/upgrade/data-migration-0.8.1-0.9.0/upgradePostgresql081to090kaaVersion.sql) files.
+2. Download the [upgradeKaa081to090.sh]({{github_url}}server/upgrade/data-migration-0.8.1-0.9.0/upgradeKaa081to090.sh), [upgradeMongo081to090KaaVersion.js]({{github_url}}server/upgrade/data-migration-0.8.1-0.9.0/upgradeMongo081to090KaaVersion.js), and [upgradePostgresql081to090kaaVersion.sql]({{github_url}}server/upgrade/data-migration-0.8.1-0.9.0/upgradePostgresql081to090kaaVersion.sql) files.
```
Comment (akokhanovskyi) [inline doc/Administration-guide/Upgrading-your-instance/0.8.1-to-0.9.0/index.md]: I suppose these should be {{github_url_raw}}

File: doc/Administration-guide/Upgrading-your-instance/0.9.x-to-0.10.x/index.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ This guide describes how to upgrade your [Kaa server]({{root_url}}Glossary/#kaa-
 Follow these steps to run the upgrade script:
 
 1. Go to your Kaa host machine.
-2. Download the [upgradeKaa090to0100.sh](https://github.com/kaaproject/kaa/blob/v0.10.0/server/upgrade/data-migration-0.9.0-0.10.0/upgradeKaa090to0100.sh) and [data-migration_0.9.0-0.10.0.jar](http://repository.kaaproject.org/repository/releases/org/kaaproject/kaa/server/upgrade/data-migration/0.10.0/data-migration_0.9.0-0.10.0.jar) to upgrade the databases.
+2. Download the [upgradeKaa090to0100.sh]({{github_url}}server/upgrade/data-migration-0.9.0-0.10.0/upgradeKaa090to0100.sh) and [data-migration_0.9.0-0.10.0.jar](http://repository.kaaproject.org/repository/releases/org/kaaproject/kaa/server/upgrade/data-migration/0.10.0/data-migration_0.9.0-0.10.0.jar) to upgrade the databases.
```
Comment (akokhanovskyi) [inline doc/Administration-guide/Upgrading-your-instance/0.9.x-to-0.10.x/index.md]: The first one should be {{github_url_raw}}.
The second link should be updated too.

**Review by akokhanovskyi — CHANGES_REQUESTED** [review]

File: doc/Administration-guide/Upgrading-your-instance/0.8.1-to-0.9.0/index.md (hunk 1 above)
Comment (vadimol) [reply doc/Administration-guide/Upgrading-your-instance/0.8.1-to-0.9.0/index.md]: Ok, let's use {{github_url_raw}} here. 
Old links referred to the HTML page on GitHub, not to a file.

Comment (akokhanovskyi) [reply doc/Administration-guide/Upgrading-your-instance/0.8.1-to-0.9.0/index.md]: That's exactly what needs to be fixed :)

File: doc/Administration-guide/Upgrading-your-instance/0.9.x-to-0.10.x/index.md (hunk 2 above)
Comment (vadimol) [reply doc/Administration-guide/Upgrading-your-instance/0.9.x-to-0.10.x/index.md]: This refers to a file in the repository. The link is supposed to be correct, however, the file is absent until we deploy Kaa 0.10.0-release.

**Review by akokhanovskyi — APPROVED** [review]

**Comment by akokhanovskyi** [comment]
@GamovCoder please review

**Review by rasendubi — APPROVED** [review]

**Review by ohamov — APPROVED** [review]

**Pull request merged**